### PR TITLE
fixed #74

### LINKED ISSUE #74: Time when the issue was open.
## Proposed Method: 
`.opened_at()`

### Description
Returns a string containing the time range when the issue was opened, like "yesterday", "last week", etc.

You can find this at 
image

 
## To what class it belongs?: 
`Issue`

### Make changes at
`src/scrape_up/github/issue.py`

### PR DESCRIPTION
**Pull request by Rubyseher**
fixed #74

### CODE REVIEW

File: src/scrape_up/github/issue.py [hunk 1]
```diff
@@ -67,4 +68,34 @@ def title(self):
...
+        if days > 7:
+            return f"{days // 7} weeks ago"
+        elif days > 0:
+            return f"{days} days ago"
+        elif seconds >= 3600:
+            return f"{seconds // 3600} hours ago"
+        elif seconds >= 60:
+            return f"{seconds // 60} minutes ago"
+        else:
+            return "just now"
```
Comment (nikhil25803) [inline src/scrape_up/github/issue.py:101]: You don't have to do that much. Just scrape the text written here in an issue.
The text after, opened this issue


image

**Review by nikhil25803 — CHANGES_REQUESTED** [review]

File: src/scrape_up/github/issue.py (hunk 1 above)
Comment (Rubyseher) [reply src/scrape_up/github/issue.py]: updated

Comment (Rubyseher) [reply src/scrape_up/github/issue.py]: updated

File: src/scrape_up/github/issue.py [hunk 2]
```diff
@@ -67,4 +66,10 @@ def title(self):
...
             message = "No title found"
             return message
-
+        
+    def opened_at(self):
+        """
+        Returns a string containing the time when the issue was opened in ISO format
+        """
+        data = self.__scrape_page()
+        return data.find('relative-time').text
```
Comment (nikhil25803) [inline src/scrape_up/github/issue.py:75]: Add the `try-exception` method to handle any unwanted error.

**Review by nikhil25803 — CHANGES_REQUESTED** [review]

File: src/scrape_up/github/issue.py (hunk 2 above)
Comment (Rubyseher) [reply src/scrape_up/github/issue.py:75]: updated

**Review by nikhil25803 — CHANGES_REQUESTED** [review]
Include this method in `documentation.md` as well.

**Review by nikhil25803 — CHANGES_REQUESTED** [review]
Include this method in `documentation.md` file. @Rubyseher.

**Comment by Rubyseher** [comment]
Okay will do it

**Comment by Rubyseher** [comment]
I have added it now

**Comment by Rubyseher** [comment]
resolved merge conflicts

**Review by nikhil25803 — APPROVED** [review]
Looks good :))

**Pull request merged**